### PR TITLE
update inputs in readme and remove duplicate or unused

### LINKED ISSUE #6: Inputs Documentation
Remove:
log_pkg_path (unused)
known_system_accounts (duplicated)

Add:
Inputs documentation to the README like RHEL7

### PR DESCRIPTION
**Pull request by em-c-rod**
resolves #6 

- [x] Removed duplicate input (known_system_accounts)
- [x] Removed unused input (log_pkg_path)
- [x] Changed input from string to number
- [x] Updated README to include all of the inputs available

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -59,43 +59,11 @@ inputs:
       - "/bin/sync"
       - "/bin/true"
 
-  # SV-230379
```
Comment (aaronlippold) [inline inspec.yml:62]: most of these are well established known system account is a RPM and RHEL based distro

File: inspec.yml [hunk 2]
```diff
@@ -59,43 +59,11 @@ inputs:
...
-      - "systemd-network"
-      - "sshd"
-      - "ntp"
-
   - name: user_accounts
     description: Accounts of known managed users
     type: Array
     value: ["vagrant"]
 
-  # SV-230379
```
Comment (aaronlippold) [inline inspec.yml:93]: Are we sure this isn't just a bad copy paste - there is a default for password reuse value right?

File: inspec.yml (hunk 1 above)
Comment (em-c-rod) [reply inspec.yml:62]: This was just a duplicate input. It exists again exactly the same later in the file

File: inspec.yml [hunk 3]
```diff
@@ -59,43 +59,11 @@ inputs:
...
-  - name: log_pkg_path
-    description: The path to the logging package
-    type: String
-    value: "/etc/rsyslog.conf"
```
Comment (aaronlippold) [inline inspec.yml:97]: logging config path default would seem to make sense generally ?

Comment (em-c-rod) [reply inspec.yml:97]: It's not used anywhere

Comment (aaronlippold) [reply inspec.yml:97]: Is it hard coded as a path and perhaps it was intended to be used but forgotten about?

Comment (em-c-rod) [reply inspec.yml:97]: This path does come up in different controls, apparently hard coded. Can we make a ticket to update that and merge this one separately?

**Review by ejaronne — APPROVED** [review]
LGTM

**Pull request merged**